### ISSUE TITLE
[BUG] qbn-dns01 not discoverable by installer

### ISSUE THREAD

**Issue by whitmell** [Qubinode/qubinode-installer#280]
**Qubinode Version**  
Qubinode release: 2.4.2  

**Describe the bug**
After running the installer for OKD4 with defaults, it can't discover qbn-dns01 the ip address

**To Reproduce**
1. ./qubinode-installer -p okd4
2. defaults for DNS/IDM server, entered Red Hat credentials
3. edited cluster setup to account for machine not meeting installer capacity requirements:
- ctrl node storage: 60 gb
- ctrl node mem: 8 gb
- compute count: 1
- compute node storage: 60 gb
- compute node mem: 8 gb
- defaults for everything else

qubinet network created
qbn-dns01 storage pool created
qbn-dns01 VM created - IP Address: Unknown

**Expected behavior**
qbn-dns01 given an IP address and discoverable by the installer

**Hardware Profile infomation**  
No result from qubinode_hardware_check.sh
OS: RHEL 8.2
Cores: 8
Mem: 24 gb
Storage: 128 gb

**Comment by tosin2013** [comment]
./qubinode-installer -p idm -d to remove idm. 

Then run the following 
./qubinode-installer -p idm
After idm deploys you can test it using
./qubinode-installer -p idm -m status 

When that competes you can rerun the okd option.
 
If you still run into issues with idm attempt a ./qubinode-installer -p idm -d then a ./qubinode-installer -m clean.  Once that is complete try and rerun the idm option with a static ip.

**Comment by whitmell** [comment]
Thanks for the suggestions.  Unfortunately I got the same result with the idm option.  Forgive my ignorance, but where would I set a static IP?  Would I need to modify one of the YAML files or can I send it as a parameter to the command?  And should I use the same IP space as my host machine?

**Comment by tosin2013** [comment]
After you delete the idm using the command `./qubinode-installer -p idm -d `  clean up the configuration using  `./qubinode-installer -m clean`  the wizard should ask you for the static ip when you are running the commands below. 

```
./qubinode-installer -m setup
./qubinode-installer -m rhsm
./qubinode-installer -m ansible
./qubinode-installer -m kvmhost
./qubinode-installer -p idm 
```

If you are still having issues I can help you in the discord group. 
https://github.com/tosin2013/qubinode-installer/blob/fixes-2.4.3/docs/communication.md
